draft-bonaventure-quic-atsss-overview.mkd

```diff
@@ -364,11 +359,11 @@ The traffic that matches an ATSSS rule can be distributed among available access
 - “Load-Balancing”: The traffic associated with the matching flow will be distributed among available access networks following a distribution ratio (e.g., 30% via a first access, 70% via a second access).
 - “Priority-based”: For this mode, accesses are assigned priority levels that indicate which access to be used first. Concretely, the traffic associated with the matching flow will be steered on the access with a high priority till congestion is detected, then the overflow will be forwarded over a low priority access. 
 
-In order to provide the above-mentioned steering modes, the ATSSS service relies upon a dedicated protocol called Performance Measurement Function (PMF) protocol. This protocol is enabled between the UE and the UPF and is meant to provide RTT measurements and reporting access availability/unavailability by the UE to the UPF. PMF is not specific to a given ATSSS Phase {{sec-phases}}. 
+In order to provide the above-mentioned steering modes, measurement information about the current network state of each path are needed. Often if a multipath capabale protocol is used, the measurements are available as part of the protocol itself. For ATSSS approaches where this is not the case, a dedicated protocol called Performance Measurement Function (PMF) protocol can be uased. This protocol is enabled between the UE and the UPF in order to provide RTT measurements and report access availability/unavailability by the UE to the UPF. 
```

Comment:
In order to provide the above-mentioned steering modes, measurement informations about the current network state of each path are needed. Often if a multipath capable protocol is used, the measurements are available as part of the protocol itself. For ATSSS approaches where this is not the case, a dedicated protocol called Performance Measurement Function (PMF) protocol can be used. This protocol is enabled between the UE and the UPF in order to provide RTT measurements and report access availability/unavailability by the UE to the UPF.